main.go

```diff
@@ -72,7 +74,19 @@ func playsound(sound []byte) {
 func (m *neko) Update() error {
 	m.count++
 	if m.state == 10 && m.count == m.min {
-		playsound(mSound["idle3"])
+		playSound(mSound["idle3"])
+	}
+	// Prevents neko from being stuck on the side of the screen
+	// or randomly travelling to another monitor
+	if m.x < 0 {
```

Comment:
Maybe we can reduce the cyclomatic complexity of this section using the new min and max internal functions in Go 1.21, what do you think?

```
m.x = max(0, min(m.x, monitorWidth))
m.y = max(0, min(m.y, monitorHeight))
```

Reply:
Sure, I didn't know about these in Go. I added a new commit.